.mk/development.mk

```diff
@@ -24,18 +24,39 @@ undeploy-loki: ## Undeploy loki.
 
 .PHONY: deploy-kafka
 deploy-kafka:
-	@echo -e "\n==> Deploy kafka"
+	@echo -e "\n==> Deploy default Kafka. Get more help on https://github.com/netobserv/documents/blob/main/kafka.md"
 	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
-	kubectl create -f "https://strimzi.io/install/latest?namespace="$(NAMESPACE) -n $(NAMESPACE)
-	kubectl create -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka-cluster.yaml" -n $(NAMESPACE)
-	kubectl create -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka-topic.yaml" -n $(NAMESPACE)
+	kubectl apply -f "https://strimzi.io/install/latest?namespace="$(NAMESPACE) -n $(NAMESPACE)
+	kubectl apply -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka/metrics-config.yaml" -n $(NAMESPACE)
+	kubectl apply -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka/default.yaml" -n $(NAMESPACE)
+	kubectl apply -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka/topic.yaml" -n $(NAMESPACE)
+
+.PHONY: deploy-kafka-tls
+deploy-kafka-tls:
+	@echo -e "\n==> Deploy Kafka with mTLS. Get more help on https://github.com/netobserv/documents/blob/main/kafka.md"
+	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
+	kubectl apply -f "https://strimzi.io/install/latest?namespace="$(NAMESPACE) -n $(NAMESPACE)
+	kubectl apply -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka/metrics-config.yaml" -n $(NAMESPACE)
+	kubectl apply -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka/tls.yaml" -n $(NAMESPACE)
+	kubectl apply -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka/topic.yaml" -n $(NAMESPACE)
+	kubectl apply -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka/user.yaml" -n $(NAMESPACE)
+	kubectl wait --timeout=180s --for=condition=ready kafkauser flp-kafka -n $(NAMESPACE)
```

Comment:
Waiting for kafka to be ready is nice.
For consistency, I think we should have both deploy-kafka target to wait.

Reply:
yeah that was especially useful here, because if kafka isn't ready while FLP is already deployed with secrets configured, pods are stuck and won't start because secret volumes can't be mounted. Even after the secrets are created, the pods are still not created  - I had to explicitly delete them.
but I agree, I'll do the same for the non-tls kafka

Reply:
done